perception_dataset/rosbag2/rosbag2_to_non_annotated_t4_converter.py

```diff
@@ -766,7 +770,23 @@ def _generate_calibrated_sensor(
                     camera_distortion=[],
                 )
             elif modality == SENSOR_MODALITY_ENUM.CAMERA.value:
-                cam_info_topic = topic_name.rsplit("/", 2)[0] + "/camera_info"
+                if self._data_type.value == "synthetic":
+                    # fix of the orientation of camera view
+                    rotation = Quaternion(
+                        rotation["w"], rotation["x"], rotation["y"], rotation["z"]
+                    )
+                    axes_fix_rotation = Quaternion(0.5, -0.5, 0.5, -0.5)
+                    rotation = rotation * axes_fix_rotation
+
+                    rotation = {
+                        "w": rotation.w,
+                        "x": rotation.x,
+                        "y": rotation.y,
+                        "z": rotation.z,
+                    }
```

Comment:
@MagdalenaKotynia 
I think this is because frame_id of image topics uses `camera_link`, but it's expected to be `camera_optical_link`.
Please see  <https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/ros-nodes/coordinate-system/#tf-tree>.

I have checked a tf tree with a command `ros2 run tf2_tools view_frames`.

[synthetic_data_xx1_with_camera.pdf](https://github.com/tier4/tier4_perception_dataset/files/12747094/synthetic_data_xx1_with_camera.pdf)
[real_data_xx1.pdf](https://github.com/tier4/tier4_perception_dataset/files/12747096/real_data_xx1.pdf)

Reply:
@yukke42 
Thank you for your comment and observation! I will consult it with the team developing AWSIM, I don't see camera_optical_link currently implemented in AWSIM. 
![image](https://github.com/tier4/tier4_perception_dataset/assets/44349533/7445add7-93a7-4116-9aed-522dfb09a0a8)

Reply:
Thanks. We can accept it now, but it will be updated after AWSIM is changed.